ui/app/manager/src/pages/page-map.ts

```diff
@@ -341,33 +370,66 @@ export class PageMap extends Page<MapStateKeyed> {
+<!--                <or-map-attribute-chart .assetId="${this._currentAsset.id}" .asset="${this._currentAsset}"></or-map-attribute-chart>-->
+            ` : ``}
             
-            <or-map id="map" class="or-map" showGeoCodingControl @or-map-geocoder-change="${(ev: OrMapGeocoderChangeEvent) => {this._setCenter(ev.detail.geocode);}}">
+
+            <or-map id="map" class="or-map" showGeoCodingControl
+                    @or-map-geocoder-change="${(ev: OrMapGeocoderChangeEvent) => {
+                        this._setCenter(ev.detail.geocode);
+                    }}" 
+            >
                 ${
-          this._assets.filter((asset) => {
-              if (!asset.attributes) {
-                  return false;
-              }
-              const attr = asset.attributes[WellknownAttributes.LOCATION] as Attribute<GeoJSONPoint>;
-              return !attr.meta || !attr.meta.hasOwnProperty(WellknownMetaItems.SHOWONDASHBOARD) || !!Util.getMetaValue(WellknownMetaItems.SHOWONDASHBOARD, attr);
-          })
-            .sort((a,b) => {
-                if (a.attributes[WellknownAttributes.LOCATION].value && b.attributes[WellknownAttributes.LOCATION].value){
-                    return b.attributes[WellknownAttributes.LOCATION].value.coordinates[1] - a.attributes[WellknownAttributes.LOCATION].value.coordinates[1];
-                } else {
-                    return;
+                        this._assets.filter((asset) => {
+                            if (!asset.attributes) {
+                                return false;
+                            }
+                            const attr = asset.attributes[WellknownAttributes.LOCATION] as Attribute<GeoJSONPoint>;
+                            return !attr.meta || !attr.meta.hasOwnProperty(WellknownMetaItems.SHOWONDASHBOARD) || !!Util.getMetaValue(WellknownMetaItems.SHOWONDASHBOARD, attr);
+                        })
+                                .sort((a, b) => {
+                                    if (a.attributes[WellknownAttributes.LOCATION].value && b.attributes[WellknownAttributes.LOCATION].value) {
+                                        return b.attributes[WellknownAttributes.LOCATION].value.coordinates[1] - a.attributes[WellknownAttributes.LOCATION].value.coordinates[1];
+                                    } else {
+                                        return;
+                                    }
+                                })
+                                .map(asset => {
+                                    return html`
+                                        <or-map-marker-asset
+                                                ?active="${this._currentAsset && this._currentAsset.id === asset.id}"
+                                                .asset="${asset}"
+                                                .config="${this.config.markers}"></or-map-marker-asset>
+                                    `;
+                                })
```

Comment:
Reformatting here makes it difficult to follow what has actually changed...we do need to enforce some linting and coding standards though